docs/_source/docs/stack_config.rst

```diff
@@ -206,15 +236,25 @@ CI/CD system like Jenkins.
 In order to use this argument, however, the role needs to have an AssumeRolePolicyDocument that
 permits the user to assume that role.
 
+**Important**: If you use set the value of the ``iam_role`` with ``!stack_output``, that `iam_role`
+will not actually be used to obtain the stack_output, but it WILL be used for all subsequent stack
+actions. Therefore, it is important that the user executing the stack have permissions to get stack
+outputs for the stack outputting the ``iam_role``.
```

Comment:
I don't use iam_role so maybe it's making it difficult for me to fully understand this.  Is this explaining a scenario for a  bootstrapping step where the admin user is using sceptre to deploy their CFN stacks for the 1st time?

Reply:
Exactly! That's exactly my own use case, actually.